mode/examples/Topics/File IO/LoadFile2/LoadFile2.pyde

```diff
@@ -0,0 +1,51 @@
+"""
+LoadFile 2
+
+This example loads a data file about cars. Each element is separated
+with a tab and corresponds to a different aspect of each car. The file stores 
+the miles per gallon, cylinders, displacement, etc., for more than 400 different
+makes and models. Press a mouse button to advance to the next group of entries.
+"""
+
+from Record import Record
+
+records = []
+record_count = 0
+num = 9  # Display this many entries on each screen.
+starting_entry = 0  # Display from this entry number
+
+def setup():
+    global records, record_count
+    size(200, 200)
+    fill(255)
+    noLoop()
+
+    body_font = loadFont("TheSans-Plain-12.vlw")
+    textFont(body_font)
+
+    lines = loadStrings("cars2.tsv")
+    for l in lines:
+        pieces = l.split(TAB)  # Load data into array
+        if len(pieces) == 9:
+            records.append(Record(pieces))
+            record_count += 1
+
+    if record_count != len(records):
+        records = records[:record_count]
```

Comment:
What do these lines do? How could record_count ever be different from len(records)?

Reply:
It makes not much sense to me either. I would guess it is some kind of guard, by the author of the original Java mode example, against spurious blank lines added to the end of the file :(

Reply:
I can remove this and/or rewrite the whole thing if you want :)

Reply:
I just don't want any broken or meaningless code in the examples.

Reply:
Please see if you like this best :)
I removed the Record class (used named tuples instead) and I decided to remove record_count (assuming the data file is properly done).